### PR TITLE
Checkout protoc module by commit hash

### PR DESCRIPTION
**Pull request by dmsolr**
1. Setup Github action for CI AND IT
2.  Using `git checkout` instead of copy source of protocol files

### CODE REVIEW

File: mock-collector/pom.xml [hunk 1]
```diff
@@ -30,11 +30,13 @@
     <artifactId>mock-collector</artifactId>
     <packaging>jar</packaging>
 
-    <name>SkyWalking-Mock-Collector</name>
+    <name>SkyWalking Mock Collector</name>
     <url>http://maven.apache.org</url>
 
     <properties>
-        <repos.protocol>https://github.com/apache/skywalking-data-collect-protocol.git</repos.protocol>
+        <protocol.version>7a22d3e</protocol.version>
```
Comment (wu-sheng) [inline mock-collector/pom.xml]: The version should be the complete commit id, not the short one.

**Comment by wu-sheng** [comment]
Please add documents to readme, and the LICENSE should be updated as you are not hosting proto now.

**Comment by kezhenxu94** [comment]
Oops, made a mistake

**Comment by wu-sheng** [comment]
Why CI doesn't exist? Something wrong?

**Comment by kezhenxu94** [comment]
> Why CI doesn't exist? Something wrong?

The workflow yaml is created in the forked repo, thus won't take effect before it's merged

File: mock-collector/bin/fetch_code.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
Comment (kezhenxu94) [inline mock-collector/bin/fetch_code.sh]: license header

**Pull request merged**